api/OffscreenCanvasRenderingContext2D.json

```diff
@@ -1100,7 +1100,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "105"
+              "version_added": false
```

Comment:
Correct! And I know the person who filed the Firefox bug 😆 
https://bugzilla.mozilla.org/show_bug.cgi?id=1228847